components/MechPage/GetInvolved.jsx

```diff
@@ -37,29 +38,38 @@ export const GetInvolved = () => (
     </h2>
 
     <div className="grid grid-cols-1 gap-8 md:grid-cols-3">
-      {list.map(({ title, desc, icon, url, urlName, isExternal }) => (
-        <div
-          key={title}
-          className="lg:p-6 flex flex-col gap-2 p-4 rounded-xl border border-l-4"
-        >
-          <div className="flex items-center">
-            {icon}
-            <h2 className="text-xl font-semibold">{title}</h2>
-          </div>
+      {list.map(
+        ({ title, desc, icon, url, urlName, isExternal, isDisabled }) => (
+          <div
+            key={title}
+            className="lg:p-6 flex flex-col gap-2 p-4 rounded-xl border border-l-4"
+          >
+            <div className="flex items-center">
+              {icon}
+              <h2 className="text-xl font-semibold">{title}</h2>
+            </div>
 
-          <p className={TEXT_CLASS}>{desc}</p>
-          {isExternal ? (
-            <a href={url} className="text-purple-600 mt-auto" target="_blank">
-              {urlName}
-              <MoveUpRight className="ml-2 inline" size={16} />
-            </a>
-          ) : (
-            <Link href={url} className="text-purple-600 mt-auto">
-              {urlName}
-            </Link>
-          )}
-        </div>
-      ))}
+            <p className={TEXT_CLASS}>{desc}</p>
+            {isDisabled ? (
+              <span>{urlName}</span>
+            ) : isExternal ? (
+              <a
+                href={url}
+                className="text-purple-600 mt-auto"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {urlName}
+                <MoveUpRight className="ml-2 inline" size={16} />
+              </a>
+            ) : (
+              <Link href={url} className="text-purple-600 mt-auto">
+                {urlName}
+              </Link>
+            )}
```

Comment:
try to avoid nested ternary operators i.e. `a ? b : b ? c : d`
they're not as easy to read / debug in future scenarios

much easier to work with something like: 

```typescript
const MyComponent ({...stuff}) => {
    if(stuff.isDisabled) return <Disabled />
    if(stuff.isExternal) return <External />
    return <>{stuff.thing}</>;
}
```

fine to leave if you're short on time however